scripts/testSpecific.js

```diff
@@ -0,0 +1,68 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+const { exec } = require('child_process');
+
+const runTests = () => {
+  const componentName = process.argv[2];
+
+  if (!componentName) {
+    console.error('Please provide a component name');
+    process.exit(1);
+  }
+
+  const commands = [
+    {
+      type: 'React Visual',
+      command: `pnpm run --filter react-workshop test --spec="**/${componentName}.*"`,
+    },
+    {
+      type: 'CSS Visual',
+      command: `pnpm run --filter css-workshop test --filter=${componentName}`,
+    },
+    {
+      type: 'Unit',
+      command: `cd ./packages/itwinui-react/src/core && pnpm test:unit --mode ${componentName} && cd ../../../../..`,
+    },
+    {
+      type: 'A11y',
+      command: `cd ./testing/a11y && pnpm cypress run --component --spec "src/SpecificComponent.cy.tsx" --env componentName=${componentName}`,
```

Comment:
`cd`ing into directories is generally unnecessary because you run these kinds of commands from the root (using [Pnpm's filter syntax](https://pnpm.io/filtering)).

Reply:
fixed this using the filter syntax you linked to